### PR TITLE
feat: added stats tab

### PR DESCRIPTION
**Pull request by dotjose**
Add Stat cards using the official PMO API https://api.pmo.gov.et/v1/patients/

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "@react-navigation/stack": "^5.3.3",
     "base-64": "^0.1.0",
     "expo": "^37.0.0",
+    "axios": "^0.19.2",
```
Comment (naody) [inline package.json:19]: No need to use axios, we're pulling all data from Firebase.

**Comment by naody** [comment]
@dotjose, I've added your stats tab implementation on master, I'm ignoring this PR do to the comments stated above. Please confirm everything you worked on is in master branch. The data source for the stats tab is now Firestore. Latest data will be pulled from the main API using cloud functions.

File: package.json (hunk 1 above)
Comment (dotjose) [reply package.json:19]: Okay, I was using the Public API to get a stat data. But if you have already on Firebase that's much better and efficient.

**Comment by dotjose** [comment]
@naody Nice work brother. It's all merged beautifully.

**Pull request closed without merging**